src/main/resources/lang/de_de.json

```diff
@@ -0,0 +1,121 @@
+	"mapping_format.proguard": "ProGuard‐Datei",
+	"type.methods": "Methoden",
+	"type.fields": "Felder",
+	"type.parameters": "Parameter",
+	"type.classes": "Klassen",
+
+	"menu.file": "Datei",
+	"menu.file.jar.open": "Java‐Archiv öffnen …",
+	"menu.file.jar.close": "Java‐Archiv schließen",
+	"menu.file.mappings.open": "Zuordnungen laden …",
+	"menu.file.mappings.save": "Zuordnungen speichern",
+	"menu.file.mappings.save_as": "Zuordnungen speichern unter …",
+	"menu.file.mappings.close": "Zuordnungen schließen",
+	"menu.file.mappings.drop": "Ungültige Zuordnungen löschen",
+	"menu.file.export.source": "Quelldateien exportieren …",
+	"menu.file.export.jar": "Java‐Archiv exportieren …",
+	"menu.file.stats": "Zuordnungsstatistiken …",
+	"menu.file.stats.title": "Enthaltene Elemente auswählen",
+	"menu.file.stats.generate": "Statistiken generieren",
+	"menu.file.exit": "Beenden",
+	"menu.decompiler": "Dekompilierer",
+	"menu.view": "Ansicht",
+	"menu.view.themes": "Farbgestaltung",
+	"menu.view.themes.default": "Standard",
+	"menu.view.themes.darcula": "Darcula",
+	"menu.view.themes.system": "System",
+	"menu.view.themes.none": "Keine (JVM‐Standard)",
+	"menu.view.languages": "Sprache",
+	"menu.view.languages.title": "Sprache ändern",
+	"menu.view.languages.summary": "Die ausgewählte Sprache wird nach dem nächsten Neustart verwendet.",
+	"menu.view.languages.ok": "Okay",
+	"menu.view.search": "Suche",
+	"menu.help": "Hilfe",
+	"menu.help.about": "Über",
+	"menu.help.about.title": "%s – Informationen",
+	"menu.help.about.ok": "Ok",
+	"menu.help.github": "GitHub‐Projekt",
+
+	"popup_menu.rename": "Umbenennen",
+	"popup_menu.javadoc": "Javadoc bearbeiten",
+	"popup_menu.inheritance": "Vererbung zeigen",
+	"popup_menu.implementations": "Implementierungen anzeigen",
+	"popup_menu.calls": "Aufrufe anzeigen",
+	"popup_menu.calls.specific": "Aufrufe anzeigen (spezifisch)",
+	"popup_menu.declaration": "Zur Deklaration gehen",
+	"popup_menu.back": "Zurück gehen",
+	"popup_menu.forward": "Vorwärts gehen",
+	"popup_menu.mark_deobfuscated": "Als deobfuskiert kennzeichnen",
+	"popup_menu.reset_obfuscated": "Zu obfuskiert zurücksetzen",
+	"popup_menu.zoom.in": "Vergrößern",
+	"popup_menu.zoom.out": "Verkleinern",
+	"popup_menu.zoom.reset": "Größe zurücksetzen",
+
+	"info_panel.classes.obfuscated": "Nicht obfuskierte Klassen:",
+	"info_panel.classes.deobfuscated": "Deobfuskierte Klassen:",
+	"info_panel.identifier": "Informationen zum Bezeichner:",
+	"info_panel.identifier.none": "Es wurde kein Bezeichner ausgewählt.",
+	"info_panel.identifier.variable": "Variable",
+	"info_panel.identifier.field": "Datenfeld",
+	"info_panel.identifier.method": "Methode",
+	"info_panel.identifier.constructor": "Konstruktur",
+	"info_panel.identifier.class": "Klasse",
+	"info_panel.identifier.type_descriptor": "Typdeskriptor",
+	"info_panel.identifier.method_descriptor": "Methodendeskriptor",
+	"info_panel.identifier.modifier": "Zugriffsmodifikator",
```

Comment:
These are attribute flags, covering quite a bit more than just access. Access specifically defines accessibility, i.e. the passive approach

Reply:
I apologize, but I cannot understand your reasoning.
I kindly ask you to further elaborate your train of thought.

Reply:
`final`, `abstract`, `static` are also examples of modifiers. There is more than access modifiers (Zugriffsmodifizierer) like `public`, `protected` and `private`

Reply:
The “Modifier:” drop-down menu does only list access modifiers:

https://pastebin.com/ApvNrAFQ
https://i.imgur.com/TBaawm6.png